dashboard/cron.py

```diff
@@ -124,26 +124,16 @@ def update_with_udw_user(self):
         # loop through multiple course ids
         for UDW_course_id in Course.objects.get_supported_courses():
 
-            # select all student registered for the course
-            user_sql = f"""select u.name AS name, 
-                        p.sis_user_id AS sis_id, 
-                        p.unique_name AS sis_name, 
-                        u.global_canvas_id AS id, 
-                        c.current_score AS current_grade, 
-                        c.final_score AS final_grade, 
-                        '{UDW_course_id}' as course_id 
-                        from user_dim u, 
-                        pseudonym_dim p, 
-                        course_score_fact c, 
-                        (select e.user_id as user_id, e.id as enrollment_id from enrollment_dim e 
-                        where e.course_id = '{UDW_course_id}' 
-                        and e.type='StudentEnrollment' 
-                        and e.workflow_state='active' ) as e 
-                        where p.user_id=u.id 
-                        and u.id = e.user_id 
-                        and c.enrollment_id =  e.enrollment_id 
-                        and p.sis_user_id is not null
-                        """
+
+            user_sql=f"""with
```

Comment:
Could you fix the indentation/formatting on this SQL? It's hard to read "as-is" compared to the old format.

Reply:
+1 for the indentation improvement.

Reply:
I fixed it. I have been lazy in organizing it